core/src/main/java/com/orange/ouds/core/component/OudsTopAppBar.kt

```diff
@@ -0,0 +1,832 @@
+ * that will have the height of [OudsTopAppBar].
+ *
+ * > Design guidelines: [unified-design-system.orange.com](https://unified-design-system.orange.com)
+ *
+ * > Design version: 1.0.0
+ *
+ * @param title The title to be displayed in the top app bar.
+ * @param modifier The [Modifier] to be applied to this top app bar.
+ * @param translucent Whether the top app bar should be translucent.
+ * @param navigationIcon The navigation icon displayed at the start of the top app bar.
+ * @param actions The actions displayed at the end of the top app bar. These can be
+ *   instances of [OudsTopAppBarAction.Icon] or [OudsTopAppBarAction.Avatar].
+ *   The default layout here is a [Row], so actions will be placed horizontally.
+ *   The maximum recommended number of actions is three. Please use a dropdown menu if you need more than three actions.
+ * @param expandedHeight This app bar's height. When a specified [scrollBehavior] causes the app bar
+ *   to collapse or expand, this value will represent the maximum height that the bar will be
+ *   allowed to expand. This value must be specified and finite, otherwise it will be ignored and
+ *   replaced with [TopAppBarDefaults.TopAppBarExpandedHeight].
+ * @param windowInsets A window insets that app bar will respect.
+ * @param scrollBehavior A [TopAppBarScrollBehavior] which holds various offset values that will be
+ *   applied by this top app bar to set up its height and colors. A scroll behavior is designed to
+ *   work in conjunction with a scrolled content to change the top app bar appearance as the content
+ *   scrolls. See [TopAppBarScrollBehavior.nestedScrollConnection].
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsTopAppBarSample
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun OudsTopAppBar(
+    title: String,
+    modifier: Modifier = Modifier,
+    translucent: Boolean = false,
+    navigationIcon: OudsTopAppBarNavigationIcon? = null,
+    actions: List<OudsTopAppBarAction> = emptyList(),
+    expandedHeight: Dp = TopAppBarDefaults.TopAppBarExpandedHeight,
+    windowInsets: WindowInsets = TopAppBarDefaults.windowInsets,
+    scrollBehavior: TopAppBarScrollBehavior? = null
+) {
+    TopAppBar(
+        title = {
+            Title(
+                title = title,
+                size = OudsTopAppBarSize.Small,
+                centerAligned = false
+            )
+        },
+        modifier = modifier.bottomBorder(),
+        navigationIcon = { navigationIcon?.Content() },
+        actions = { actions.forEach { it.PolymorphicContent() } },
+        expandedHeight = expandedHeight,
+        windowInsets = windowInsets,
+        colors = colors(translucent = translucent),
+        scrollBehavior = scrollBehavior
+    )
+}
+
+//TODO update DSM link when available
+/**
+ * Top app bars display information and actions at the top of a screen.
+ *
+ * This small top app bar has a header title that is horizontally aligned to the center.
```

Comment:
Should we add that this app bar has also slots for a title, navigation icon, and actions like the other ones?